pkg/util/util.go

```diff
@@ -161,3 +168,37 @@ func GetMapKeysAsString(m map[string][]byte) (keys string) {
 
 	return
 }
+
+// AtLeast_v1_25 returns true if the RKE2 version is at least v1.25.0.
+func AtLeast_v1_25(rke2version string) (bool, error) {
```

Comment:
```suggestion
func AtLeastv125(rke2version string) (bool, error) {
```
or 
```suggestion
func greaterThanV125(rke2version string) (bool, error) {
```

Snake case is not idiomatic in golang

Reply:
First suggestion implemented